A66-otel-stats.md

```diff
@@ -0,0 +1,325 @@
+# OpenTelemetry Metrics
+
+*   Author: Yash Tibrewal (@yashykt)
+*   Approver: Mark Roth (@markdroth)
+*   Status: In Review
+*   Implemented in: <language, ...>
+*   Last updated: Jul 20, 2023
+*   Discussion at: https://groups.google.com/g/grpc-io/c/po-deqYEQzE
+
+## Abstract
+
+Propose a metrics data model for gRPC OpenTelemetry metrics.
+
+## Background
+
+There are a collection of
+[metrics](https://github.com/census-instrumentation/opencensus-specs/blob/master/stats/gRPC.md)
+proposed by OpenCensus for gRPC. OpenCensus is no longer being actively
+maintained and is being
+[deprecated](https://opentelemetry.io/blog/2023/sunsetting-opencensus/#:~:text=Compatibility%20specification%204.-,What%20to%20Expect%20After%20July%2031st%2C%202023,found%20will%20not%20be%20patched.),
+with OpenTelemetry suggested as the successor framework.
+
+### Related Proposals:
+
+*   [A45: Exposing OpenCensus Metrics and Tracing for gRPC retry](A45-retry-stats.md)
+
+## Proposal
+
+### Metrics Schema
+
+#### Units
+
+Following the
+[OpenTelemetry Metrics Semantic Conventions](https://opentelemetry.io/docs/specs/otel/metrics/semantic_conventions/),
+the following units are used -
+
+*   Latencies are measured in float64 seconds, `s`
```

Comment:
Might want to be more specific about which part of the OpenTelemetry Metrics Semantic Conventions this follows; 
- [http.server.duration is specified as seconds (s)](https://opentelemetry.io/docs/specs/otel/metrics/semantic_conventions/rpc-metrics/#rpc-server)
- [rpc.server.duration is specified as milliseconds (ms)](https://opentelemetry.io/docs/specs/otel/metrics/semantic_conventions/rpc-metrics/#rpc-server)